layouts/shortcodes/gke/latest-version.html

```diff
@@ -1 +1 @@
-1.5.0
\ No newline at end of file
+1.5.1
```

Comment:
We need to wait for this release to be made on https://github.com/kubeflow/gcp-blueprints/releases before merging this doc change.